crates/uv-virtualenv/src/virtualenv.rs

```diff
@@ -276,81 +277,94 @@ pub(crate) fn create(
+                        interpreter,
+                        &base_python,
+                        &scripts,
+                        python_home,
+                    )?;
 
-            if interpreter.markers().implementation_name() == "pypy" {
-                copy_launcher_windows(
-                    WindowsExecutable::PythonMajor,
-                    interpreter,
-                    &base_python,
-                    &scripts,
-                    python_home,
-                )?;
-                copy_launcher_windows(
-                    WindowsExecutable::PythonMajorMinor,
-                    interpreter,
-                    &base_python,
-                    &scripts,
-                    python_home,
-                )?;
-                copy_launcher_windows(
-                    WindowsExecutable::PyPy,
-                    interpreter,
-                    &base_python,
-                    &scripts,
-                    python_home,
-                )?;
-                copy_launcher_windows(
-                    WindowsExecutable::PyPyMajor,
-                    interpreter,
-                    &base_python,
-                    &scripts,
-                    python_home,
-                )?;
-                copy_launcher_windows(
-                    WindowsExecutable::PyPyMajorMinor,
-                    interpreter,
-                    &base_python,
-                    &scripts,
-                    python_home,
-                )?;
-                copy_launcher_windows(
-                    WindowsExecutable::PyPyw,
-                    interpreter,
-                    &base_python,
-                    &scripts,
-                    python_home,
-                )?;
-                copy_launcher_windows(
-                    WindowsExecutable::PyPyMajorMinorw,
-                    interpreter,
-                    &base_python,
-                    &scripts,
-                    python_home,
-                )?;
+                }
+                "pypy" => {
+                    // For PyPy, copy all versioned executables and all PyPy-specific executables.
+                    copy_launcher_windows(
+                        WindowsExecutable::PythonMajor,
+                        interpreter,
+                        &base_python,
+                        &scripts,
+                        python_home,
+                    )?;
+                    copy_launcher_windows(
+                        WindowsExecutable::PythonMajorMinor,
+                        interpreter,
+                        &base_python,
+                        &scripts,
+                        python_home,
+                    )?;
+                    copy_launcher_windows(
+                        WindowsExecutable::Pythonw,
+                        interpreter,
+                        &base_python,
+                        &scripts,
+                        python_home,
+                    )?;
```

Comment:
Seems wrong that we omit `PythonMajorw` and `PythonMajorMinorw`, but that's the existing behavior.